Cabal-tests/tests/ParserTests/regressions/decreasing-indentation.check

```diff
@@ -0,0 +1,16 @@
+decreasing-indentation.cabal:40:3: Inconsistent indentation at lines 40, 41, 42, 43, 44, 50, 52, 57, 58, 70, 71
```

Comment:
Maybe `40-44, 50, 52, 57-58, 70-71` would be clearer?

Reply:
I changed the code to report the warning only at lines where indentation jumps, so now it is `Indentation jumps at lines 40, 50, 57, 58, 70` which is more to the point.